dockerfile

```
@@ -21,8 +21,11 @@ RUN cp /etc/nuoj/setting/setting.json /etc/nuoj/setting.json
 
 # install tailwindcss and react.js from npm
 WORKDIR /etc/nuoj
-RUN npm install -D tailwindcss
-RUN npm install babel-cli@6 babel-preset-react-app@3
+# RUN npm install -D tailwindcss
+# RUN npm install babel-cli@6 babel-preset-react-app@3
+RUN npm install
+RUN npx msw init src/js --save
```

可以考慮使用 [pnpm](https://pnpm.io/zh-TW/)

```suggestion
RUN npm install pnpm
RUN pnpm install 
RUN npx msw init src/js --save
```